lte/cloud/go/services/subscriberdb_cache/worker.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscriberdb_cache
+
+import (
+	"time"
+
+	"magma/lte/cloud/go/services/subscriberdb"
+	"magma/lte/cloud/go/services/subscriberdb/storage"
+	"magma/orc8r/cloud/go/clock"
+	"magma/orc8r/cloud/go/services/configurator"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"github.com/thoas/go-funk"
+)
+
+func MonitorDigests(store storage.DigestLookup, config Config) {
+	for {
+		generatedDigestsByID, err := RenewDigests(store, config)
+		if err != nil {
+			glog.Errorf("Error monitoring digests: %+v", err)
+		}
+		glog.Infof("Generated digests per network: %+v", generatedDigestsByID)
+
+		// Set interval between function calls to prevent spewing of error messages
+		time.Sleep(time.Duration(config.SleepIntervalSecs) * time.Second)
+	}
+}
+
+// RenewDigests continuously monitors changes in network configs and keeps a
+// list of up-to-date digests per network, updated at a configurable interval,
+// in the db store.
+//
+// NOTE: This function is exported for testing purposes. Call MonitorDigests
+// instead for conventional usage.
+func RenewDigests(store storage.DigestLookup, config Config) (map[string]string, error) {
+	networksToRenew, networksToRemove, err := getNetworksToUpdate(store, config.UpdateIntervalSecs)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Get networks to update")
+	}
+	err = store.DeleteDigests(networksToRemove)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Remove invalid networks")
+	}
+
+	generatedDigestsByID := map[string]string{}
+	for _, networkID := range networksToRenew {
```

Comment:
So, an error from a single network shouldn't cause all the other networks to be skipped. So instead of early-returning, use multierror -- e.g. https://github.com/magma/magma/blob/46e4f80dcd6ddd7b100c8fcd4532b5a60fba85f4/orc8r/cloud/go/parallel/map.go#L71-L83